.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: go
+
+go:
+  - "1.10"
+
+install:
+  - echo "install fzp tool"
+  - go get github.com/fritzing/fzp/bin/fzp
+  - echo "fzp succefully installed. Version:"
+  - fzp -v
+  - go get github.com/fritzing/fzb/bin/fzb
```

Comment:
why are you doing this twice?

Reply:
Yes, the naming is a bit dangerous. It is fz**p** and fz**b**, so two different validators

Reply:
oh, sorry about that.